servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -209,7 +211,7 @@ public EvictionPlan freeSpace(long bytesToBeAvailable, BlockStoreLocation locati
       }
     }
 
-    return EvictorUtils.legalCascadingPlan(bytesToBeAvailable, plan, mMetaManager) ? plan : null;
+    return EvictorUtils.validCascadingPlan(bytesToBeAvailable, plan, mMetaManager) ? plan : null;
```

Comment:
@apc999 I'll remove both this validation and the verification of block existence above in PR: https://github.com/amplab/tachyon/pull/1097